client/my-sites/email/email-management/titan-manage-mailboxes/index.jsx

```diff
@@ -101,69 +102,74 @@ class TitanManageMailboxes extends Component {
 
 				{ selectedSite && hasSubscription && <QuerySitePurchases siteId={ selectedSite.ID } /> }
 
-				<DocumentHead title={ translate( 'Manage all mailboxes' ) } />
+				<Main wideLayout={ true }>
+					<DocumentHead title={ translate( 'Manage all mailboxes' ) } />
 
-				<EmailHeader currentRoute={ currentRoute } selectedSite={ selectedSite } />
+					<EmailHeader currentRoute={ currentRoute } selectedSite={ selectedSite } />
 
-				<HeaderCake onClick={ this.handleBack }>{ translate( 'Manage all mailboxes' ) }</HeaderCake>
+					<HeaderCake onClick={ this.handleBack }>
+						{ translate( 'Manage all mailboxes' ) }
+					</HeaderCake>
 
-				<EmailPlanHeader
-					domain={ domain }
-					hasEmailSubscription={ hasSubscription }
-					isLoadingPurchase={ isLoadingPurchase }
-					purchase={ purchase }
-					selectedSite={ selectedSite }
-				/>
-
-				<VerticalNav>
-					<VerticalNavItemEnhanced
-						description={ translate(
-							'View settings required to configure third-party email apps'
-						) }
-						external={ true }
-						materialIcon="dvr"
-						path={ this.getPath( TITAN_CONTROL_PANEL_CONTEXT_CONFIGURE_DESKTOP_APP ) }
-						text={ translate( 'Configure desktop app' ) }
+					<EmailPlanHeader
+						domain={ domain }
+						hasEmailSubscription={ hasSubscription }
+						isLoadingEmails={ false }
```

Comment:
Passing `false` for this particular prop makes me feel that something is not quite right. I know I'm the one who originally built that page but now I'm thinking we should retrieve and pass `emailAccount` to this `EmailPlanHeader` component.

Reply:
Agreed -- we definitely have some cleanup to handle around the data and wiring.